_posts/2023-08-01-personas-framework.md

```diff
@@ -0,0 +1,50 @@
+
+## Research study overview
+
+The OpenSearch Project generally requests community participation in UX research through deep-dive sessions. For this study, we recruited search users and experts from the 2023 Haystack Conference. In the deep-dive sessions with these users, we discussed various topics relevant to users’ data use cases, including their end user or customer needs, motivations, current search solutions, pain points, and paths to task completion for setting up search solutions. We also developed a user flow that we modified iteratively to identify successful user outcomes.
+
+At Berlin Buzzwords 2023, we shared insights from the deep-dive sessions held during BarCamp, a session format that is less formal than a speaker presentation. The session agendas are set at the beginning of the session, and the topics are voted on, selected, and then discussed. In our session, “Search Users and Jobs to Be Done,” we presented on who we determined the search producer to be. We also presented tasks and workflows that we had determined to be essential to producers based on the user and expert interviews. The audience comprised search engineers and search solution providers who represented the core personas and focus of our design. Audience feedback was provided in a focus group format. The audience offered details of the nuances and variations of their use cases. They also provided viewpoints on how to rethink the proposed flow based on their need to create real-world solutions. Audience input was useful in clarifying the persona of the core search producer and identifying other users to consider as we develop our search persona framework. 
+
+## Key takeaways from the study
+
+The following are key takeaways from our research study:
+
+1. The search producer persona could be a search relevance engineer or a rater (a human judge and domain expert who rates the relevance of particular queries).
+2. The search consumer persona could be the end user who performs the search or the product manager who sets expectations for the search solution.
+
+## Conclusion
+
+Users who set up search solutions must accomplish four pivotal tasks: (1) discover and understand data, (2) understand the end user, (3) describe a successful search solution, and (4) tailor the end user experience. 
+
+The research findings help articulate the main persona framework for search, and ongoing research continues to give us a deeper understanding of the search user and the variations of domain-specific user dependencies and user roles. Fleshing out a comprehensive set of personas will help us design more meaningful user experiences. We encourage our search users and community members to continue providing feedback on the persona framework. We look forward to hearing from you! 
```

Comment:
There should ideally be a word after the first instance of "search". "search users"? "search solutions"? Or it could perhaps read "the main persona framework for the OpenSearch Project".

Reply:
@natebower Revised. It should be "solutions," to align to the intro  para. Thanks for your patience :)